actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -641,6 +639,22 @@ private void processUserRequest(
     TelemetryUtil.telemetryProcessingCall(userMap, targetObject, correlatedObject);
   }
 
+  private void setStateValidation(Map<String, Object> userMap) {
+    String rootOrgId = (String) userMap.get(JsonKey.ROOT_ORG_ID);
+    String custodianRootOrgId = getCustodianRootOrgId();
+    if(!rootOrgId.equals(custodianRootOrgId)) {
+      userMap.put("isStateValidated", true);
```

Comment:
use constant or jsonKey for isStateValidated

Reply:
done